### PR TITLE
Feat : added Image and input components

### PR DESCRIPTION
**Pull request by MurtazaBagwala07**

### CODE REVIEW

**Review by rohit8686 — APPROVED** [review]
Looks good !

File: components/images/images.css [hunk 1]
```diff
@@ -0,0 +1,18 @@
+.image-container{
+    max-width: 100%;
+    display: inline-block;
+}
+
+.image-resp{
+    max-width: 100%;
+    height: auto
```
Comment (Rohanmond) [inline components/images/images.css]: put a semicolon

**Review by Rohanmond — APPROVED** [review]
everything looks good

File: components/images/images.css [hunk 2]
```diff
@@ -0,0 +1,18 @@
+.image-container{
+    max-width: 100%;
+    display: inline-block;
```
Comment (anuj-kumary) [inline components/images/images.css:3]: No need to write display:inline-block

**Review by anuj-kumary — CHANGES_REQUESTED** [review]
Also in the input component, you have to add textarea input also.

File: components/images/images.css (hunk 1 above)
Comment (MurtazaBagwala07) [reply components/images/images.css]: done

File: components/images/images.css (hunk 2 above)
Comment (MurtazaBagwala07) [reply components/images/images.css:3]: will remove it.

**Comment by MurtazaBagwala07** [comment]
> Also in the input component, you have to add textarea input also.

ok , on it

**Pull request merged**